### DISCUSSION TITLE
Macro doesn't want to run with FreeCad 0.20.2?

### DISCUSSION THREAD

**Discussion by SimonDMiller** [LubomirJagos/FreeCAD-OpenEMS-Export#25] [Q&A] [answered]
I know they did a lot of work on the Addon Manager during the 0.20 release but not sure if the macros are expected to work without change. I'm following the video on Youtube showing the install and while I can get the Macro Window to show the Free-CAD-OpenEMS-Export.FCMacro as a choice when I set the macro location to the unzipped file as directed it doesn't open the GUI. Instead I get this error:

01:36:02  Traceback (most recent call last):
  File "C:/Users/sdmil/Downloads/FreeCAD-OpenEMS-Export-main/FreeCAD-OpenEMS-Export-main/FreeCAD-OpenEMS-Export.FCMacro", line 25, in <module>
    FOE_spec.loader.exec_module(FOE_mod)
  File "<frozen importlib._bootstrap_external>", line 844, in exec_module
  File "<frozen importlib._bootstrap_external>", line 980, in get_code
  File "<frozen importlib._bootstrap_external>", line 1037, in get_data
<class 'FileNotFoundError'>: [Errno 2] No such file or directory: 'C:\\Users\\sdmil\\Downloads\\FreeCAD-OpenEMS-Export-main\\FreeCAD-OpenEMS-Export-main\\FreeCAD-OpenEMS-Export/ExportOpenEMSDialog.py'

Am I doing something wrong, or do I just need to downgrade to some version of 0.19? 

Thanks
Simon

**Comment by LubomirJagos** [comment, marked as answer]
Hi, thanks for downloading, now looking at it, best it to download release version, but doesn't matter, that file is starting point which is supposed to be copied outside your downloaded folder into parent, reason is that you should place this macro folder into folder where others macros are located and have that startpoint along with your others macro. There is loading line:
    FreeCAD-OpenEMS-Export/ExportOpenEMSDialog.py
    
So solution is:
    solution 1. rename folder with this addod as FreeCAD-OpenEMS-Export and place that FreeCAD-OpenEMS-Export.FCMacro into parent folder to and load it in FreeCAD
    solution 2. in FreeCAD open as macro file ExportOpenEMSDialog.py (this is same what that startpoint macro will do)
